runtimes/bulletin-westend/tests/tests.rs

```diff
@@ -178,6 +201,65 @@ fn transaction_storage_runtime_sizes() {
 	});
 }
 
+/// Test maximum write throughput: 8 transactions of 1 MiB each in a single block (8 MiB total).
+#[test]
+fn transaction_storage_max_throughput_per_block() {
+	use bulletin_westend_runtime as runtime;
+	use bulletin_westend_runtime::BuildStorage;
+	use frame_support::assert_ok;
+	use pallet_transaction_storage::{AuthorizationExtent, Call as TxStorageCall};
+	use sp_keyring::Sr25519Keyring;
+
+	const NUM_TRANSACTIONS: u32 = 8;
+	const TRANSACTION_SIZE: usize = 1024 * 1024; // 1 MiB
+	const TOTAL_BYTES: u64 = (NUM_TRANSACTIONS as u64) * (TRANSACTION_SIZE as u64); // 8 MiB
+
+	sp_io::TestExternalities::new(
+		runtime::RuntimeGenesisConfig::default().build_storage().unwrap(),
+	)
+	.execute_with(|| {
+		let account = Sr25519Keyring::Alice;
+		let who: AccountId = account.to_account_id();
+
+		// fund Alice to cover length-based tx fees
+		let initial: Balance = 10_000_000_000_000_000_000u128;
+		<pallet_balances::Pallet<Runtime> as FungibleMutate<_>>::set_balance(&who, initial);
```

Comment:
Is this really needed?
```suggestion
```

Reply:
fixed partially by: https://github.com/paritytech/polkadot-bulletin-chain/pull/148/commits/3f9f609631de0810e9ca3c912c978d3c8264c7ed